### PR TITLE
1729 load spinner if pending balance

### LINKED ISSUE #1729: 🐛 [Bug] load spinner if pending balance
## 🐛 Bugreport
Immediately after logging in, the message "You have no transactions on your account yet" appears. This sometimes lasts a few (shock) seconds.
A better message would be "Get transactions" or similar.
or a spinner wheel.
If it is difficult to distinguish between the states, I would simply let the spinner run for 3 seconds and then display the rest.

https://user-images.githubusercontent.com/1324583/161422805-ec78f5d2-c40c-4d80-a62b-45001d15cea0.mp4

### PR DESCRIPTION
**Pull request by ogerly**
## 🍰 Pullrequest
an animated icon is displayed if no balance has been loaded yet. this affects the balance in the navbar, the balnaceim status and the start text in the overview. 

### Issues
- fixes #1729 

https://user-images.githubusercontent.com/1324583/161424839-b5d56ee9-c204-4708-abc5-11c0a8396217.mp4

### CODE REVIEW

File: frontend/src/components/Status.spec.js [hunk 1]
```diff
@@ -37,6 +37,9 @@ describe('Status', () => {
           pending: false,
         })
       })
+      it('it no displays an animation icon test-pending-icon', () => {
```
Comment (Mogge) [inline frontend/src/components/Status.spec.js:40]:
```suggestion

      it('does not display an animation icon test-pending-icon', () => {
```

File: frontend/src/components/Status.spec.js [hunk 2]
```diff
@@ -26,8 +26,8 @@ describe('Status', () => {
     })
 
     describe('balance is pending', () => {
-      it('it displays an en-dash', () => {
-        expect(wrapper.find('div.gdd-status-div').text()).toEqual('em-dash GDD')
+      it('it displays an animation icon test-pending-icon', () => {
```
Comment (Mogge) [inline frontend/src/components/Status.spec.js:29]:
```suggestion
      it('displays an animation icon test-pending-icon', () => {
```

**Review by Mogge — CHANGES_REQUESTED** [review]

File: frontend/src/components/Menu/Navbar.vue [hunk 3]
```diff
@@ -10,7 +10,10 @@
       </div>
 
       <b-navbar-nav class="ml-auto" is-nav>
-        <b-nav-item>{{ pending ? $t('em-dash') : balance | amount }} {{ $t('GDD') }}</b-nav-item>
+        <b-nav-item>
+          <b-icon v-if="pending" icon="three-dots" animation="cylon"></b-icon>
+          <div v-else>{{ pending ? $t('em-dash') : balance | amount }} {{ $t('GDD') }}</div>
```
Comment (ulfgebhardt) [inline frontend/src/components/Menu/Navbar.vue:15]: no - why? remove?

File: frontend/src/components/Status.vue [hunk 4]
```diff
@@ -1,7 +1,10 @@
 <template>
   <div class="gdd-status">
     <div class="p-0 gdd-status-div">
-      {{ pending || balance === null ? $t('em-dash') : $n(balance, 'decimal') }} {{ statusText }}
+      <b-icon v-if="pending" icon="three-dots" animation="cylon" class="test-pending-icon"></b-icon>
+      <div v-else>
+        {{ pending || balance === null ? $t('em-dash') : $n(balance, 'decimal') }} {{ statusText }}
```
Comment (ulfgebhardt) [inline frontend/src/components/Status.vue:6]: ??? why pending?

**Review by ulfgebhardt — CHANGES_REQUESTED** [review]

**Pull request merged**